### ISSUE TITLE
SDAF Deployer preparation should have a choice from where to obtain the code

### ISSUE THREAD

**Issue by enterprise-1701-d** [Azure/sap-automation#257]
I'm using SDAF with Azure DevOps and the pipelines. So I think, that my codebase is the one I have cloned into the Azure DevOps repository.
Here I control whether I sync with the code on GitHub or not, because I want to have a stable environment not being surprised by code changes in GitHub.

During resolution of errors regarding the Azure DevOps based setup of the deployer I found out, that the configure_deployer.sh is obtaining the repository from GitHub and not from the Azure DevOps-Repo.

if you check configure_deployer.sh you find

`# Azure SAP Automated Deployment directories`
`asad_home="${HOME}/Azure_SAP_Automated_Deployment"`
`asad_ws="${asad_home}/WORKSPACES"`
`asad_repo="https://github.com/Azure/sap-automation.git"`
`asad_dir="${asad_home}/$(basename ${asad_repo} .git)"`

and later
`#`
`# Clone Azure SAP Automated Deployment project repository`
`#`
`if [[ ! -d "${asad_dir}" ]]; then`
`    git clone "${asad_repo}" "${asad_dir}"`
`fi`

Bottom line, It would be great to make that fact aware in the variable group SDAF-MGMT  used in pipeline "Control plane deployment"  and add an additional variable which is than handed over from the pipeline to the script and people can decide from where to obtain the code for the deployer configuration.

Thanks
Jakob Sauer
